deployment/network-operator/templates/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -31,6 +31,7 @@ spec:
     repository: {{ .Values.nvPeerDriver.repository }}
     version: {{ .Values.nvPeerDriver.version }}
     gpuDriverSourcePath: {{ .Values.nvPeerDriver.gpuDriverSourcePath }}
+    ofedDriverSourcePath: {{ .Values.nvPeerDriver.ofedDriverSourcePath }}
```

Comment:
you already have the .Values.ofedDriver.deploy which indicate if we deploy OFED so why we need this options as well?

Reply:
I agree. We can remove this option from helm values. That is not what we want to expose right now. I will update PR